Watchman.Web/HangfireJobsService.cs

```diff
@@ -20,11 +20,11 @@ public void SetDefaultJobs(IContainer container)
             var generators = new List<(ICyclicService, RefreshFrequent, bool shouldTriggerNow)>
             {
                 (container.Resolve<MessagesService>(), RefreshFrequent.Quarterly, true),
+                (container.Resolve<ServerMessagesCacheService>(), RefreshFrequent.Quarterly, false),
```

Comment:
zastanawiam sie nad podpięciem jakiegoś headless cms (przykładowo strapi jest całkiem przyjemny, ale nie wiem jak z jego granicami możliwości, czy nie będą za małe)

tymczasowo, zanim nasz panel zarządzania nie będzie gotowy

do zarządzania konfiguracjami itp 
czyli wszelkie odpowiedzi bota
wszelkie konfiguracje
wszelkie CustomCommands

powinniśmy mieć dzięki temu szybciej to czego potrzebujemy
a nawet dużo szybciej

problem jest taki że trzeba będzie siąść do wyklikiwania konfiguracji + ogarnąć integracje

(komentarz nie dotyczy tego kodu, po prostu komentarze przypięte do kodu są lepiej widoczne od tych pisanych do całego pull requesta, bo github ma do nich taki sobie UI, nie można ich komentować itd)